src/test/java/org/eolang/speco/MainTest.java

```diff
@@ -33,11 +37,37 @@
  */
 public final class MainTest {
 
+    /**
+     * Relative path to the directory with tests.
+     */
+    private final String tests = "./src/test/resources/";
+
+    /**
+     * Relative path to the directory with input files.
+     */
+    private final String input = this.tests.concat("in");
```

Comment:
@advasileva it's better to use `Path.resolve()` instead of concatenating strings (since this is not platform independent)